all.json

```
@@ -212,6 +212,7 @@
     "polkastarter.ws",
     "polkastartor.com",
     "polkastater.link",
+    "polkastarter.walletconnect-help.com",
```

```suggestion
    "polkastarter.walletconnect-help.com",
    "walletconnect-help.com",
```

Let's just ban top-level as well. (Unsorted, CI will take care of it)